source/client/HandView.py

```diff
@@ -96,6 +99,8 @@ def nextEvent(self):
                 elif self.sort_btn.isOver(pos):
                     self.hand_info.sort(key=lambda wc: wc.key)
                     self.hand_info = self.refreshXY(self.hand_info)
+                elif self.ready_btn.isOver(pos):
+                    print('clicked on ready button')
```

Comment:
If you can leave a TODO for me so I can find this more easily I would appreciate it. 
Something like TODO: Helen - make call to controller here!